packages/owl-bot/src/config-files.ts

```diff
@@ -67,45 +67,89 @@ export interface OwlBotYaml {
 // The default path where .OwlBot.yaml is expected to be found.
 export const DEFAULT_OWL_BOT_YAML_PATH = '.github/.OwlBot.yaml';
 
-function validatePath(path: string, fieldName: string) {
-  if (path && path[0] !== '/') {
-    throw `I expected the first character of ${fieldName} to be a '/'.  Instead, I found ${path}.`;
+function validatePath(
+  path: string,
+  fieldName: string,
+  errorMessages: string[]
```

Comment:
I see how this code reports all the issues in a config file, but I don't see how it transforms the error message `[object Object]` into something more informative.

If line 72 were the error being thrown, then we would have seen the error message, not `[object Object]`.

Reply:
This wasn't the `[object Object]` error message. The `throw validate.errors` from old line 101 was the `[object Object]` message (it was `ErrorObject<string, Record<string, any>, unknown>` from `ajv`). After this change, we collect better error messages and put into the `InvalidOwlBotConfigError` that is thrown for a better message (new lines 114-120).

We change this helper to add error messages to a list to make types easier (collecting error messages than collecting various Error objects).